src/openrct2-ui/windows/Player.cpp

```diff
@@ -169,15 +186,22 @@ class PlayerWindow final : public Window
 
     void OnMouseUp(WidgetIndex widgetIndex) override
     {
+        switch (widgetIndex)
+        {
+            case WIDX_CLOSE:
+                Close();
+                break;
```

Comment:
return;

Reply:
Makes sense, thanks - will push the changes now!